android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetConferenceOptions.java

```diff
@@ -105,6 +110,17 @@ public Builder setRoom(String room) {
             return this;
         }
 
+        /**
+         * Sets the conference title.
```

Comment:
Don't you wanna call it "conference subject" instead?

Reply:
indeed